documentation/specs/custom-cultures.md

```diff
@@ -0,0 +1,36 @@
+# MSBuild Custom Cultures Support
+
+## Overview
+
+The `MSBUILDENABLECUSTOMCULTURES` feature flag provides an opt-in mechanism for handling custom culture-specific resources in MSBuild projects. This feature allows for greater control over which directories are treated as culture-specific resources during the build process.
+
+## Purpose
```

Comment:
I'd add a sentence about why people want custom cultures in the first place since this is a nontrivial spec.